src/styles/components/weather.scss

```diff
@@ -1,3 +1,18 @@
 .weather {
   color: $primary-text-color;
+
+  &__text {
+    font-weight: bold;
+  }
+  &__info {
+    flex-direction: column;
+    justify-content: space-evenly;
+    display: flex;
+  }
+  &__cold {
```

Comment:
Being pedantic, but under BEM this would be a modifier

[BEM](http://getbem.com/
)

[How to use](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)

Reply:
So something like &--cold {}, as we are _modifying_ the element